lib/vanagon/component.rb

```diff
@@ -92,6 +92,15 @@ class Component
     # preinstall_actions is a two-dimensional Array, describing scripts that
     # should be executed before a given component is installed.
     attr_accessor :preinstall_actions
+    # install_triggers is a two-dimensional Array, describing scripts that
+    # should be executed when a package is installed or upgraded
+    attr_accessor :install_triggers
```

Comment:
Isn't `@install_triggers` actually a one-dimensional array of OpenStructs?

Reply:
Yup, good catch, that was a copypasta error